plugins/meta/bandwidth/main.go

```diff
@@ -101,14 +102,16 @@ func getBandwidth(conf *PluginConf) *BandwidthEntry {
 	return conf.BandwidthEntry
 }
 
-func validateRateAndBurst(rate int, burst int) error {
+func validateRateAndBurst(rate, burst uint64) error {
 	switch {
 	case burst < 0 || rate < 0:
 		return fmt.Errorf("rate and burst must be a positive integer")
 	case burst == 0 && rate != 0:
 		return fmt.Errorf("if rate is set, burst must also be set")
 	case rate == 0 && burst != 0:
 		return fmt.Errorf("if burst is set, rate must also be set")
+	case burst/8 > math.MaxUint32:
```

Comment:
Rate can be more than 4 GB here?

Reply:
nit: Could you help to add some extra tests for `validateRateAndBurst`?

Reply:
> Rate can be more than 4 GB here?

Can be as it comes in argument as uint64 but later gets converted to a byte and casted to uint32. Kept the same logic in this validity check as is used later down the path for call to tc.

Reply:
OK, got your point.

Reply:
This new case deserves for new test case. Can you add it in this PR?

Reply:
Yes, just need to check out the testing framework, will try to make some time tomorrow.